## Lesson 5 ##

### Task 1 ###

Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256

#### Решение ####

In [4]:
M = 80
n = 256
S = 16 ** (1/2)
S

4.0

In [6]:
A_2 = 0.95 + (1-0.95)/2
A_2

0.975

In [3]:
Z_a_2 = 1.96

In [7]:
md = Z_a_2 * S/(n ** (1/2))
md

0.49

In [8]:
X1 = M-md
X1

79.51

In [10]:
X2 = M+md
X2

80.49

Ответ: [79.51, 80.49]

### Task 2 ###

В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: (6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1). Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

#### Решение ####

In [24]:
import numpy as np

In [12]:
x = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
x

array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [15]:
M = x.sum()/len(x)
M

6.590000000000001

In [16]:
S = (np.sum((x - M)**2)/(len(x)-1)) ** (1/2)
S

0.4508017549014448

In [35]:
A_2 = (1 - 0.95)

In [20]:
T_a_2 = 2.262

In [21]:
md = T_a_2 * S/(n ** (1/2))
md

0.06373209809919175

In [22]:
X1 = M-md
X1

6.526267901900809

In [23]:
X2 = M+md
X2

6.6537320980991925

Ответ: [6.53, 6.65]

### Task 3 ###

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

#### Решение ####

In [26]:
M0 = 17
M1 = 17.5
S = 4 ** (1/2)
n = 100

In [25]:
Z_a = 1.645

In [28]:
Z_n = (M1 - M0)/(S / (n ** (1/2)))
Z_n

2.5

Ответ: Z_n > Z_a => гипотеза неверна

### Task 4 ###

Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек.
Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно,
что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%

In [36]:
A = 0.01

In [ ]:
T_a = 2.2821

In [31]:
M0 = 200

In [29]:
x = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
x

array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

In [32]:
M1 = x.sum()/len(x)
M1

198.5

In [34]:
S = (np.sum((x - M1)**2)/(len(x)-1)) ** (1/2)
S

4.453463071962462

In [38]:
T_n = (M1 - M0)/(S / (n ** (1/2)))
T_n

-3.368165348543039

Ответ: поскольку нормальное распределение симметрично, то можно сравнивать T_a и T_n по модулю, |T_n| > |T_a| => гипотеза неверна